packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -158,7 +158,7 @@ const BulletinSummary = styled.p`
   padding: 0 ${GEL_SPACING};
   @media(min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     padding-left: 0;
-    padding-right: ${GEL_SPACING};
+    padding-right: ${({ type }) => (type === 'audio' ? `${GEL_SPACING}` : `0`)};
```

Comment:
The browser will not render the rule, leaving an empty string.
```suggestion
    padding-right: ${({ type }) => (type === 'audio' ? `${GEL_SPACING}` : ``)};
```